src/ray/common/status.h

```diff
@@ -101,6 +101,9 @@ enum class StatusCode : char {
   ObjectAlreadySealed = 23,
   ObjectStoreFull = 24,
   TransientObjectStoreFull = 25,
+  // grpc status
+  GrpcUnavailable = 26,
```

Comment:
Why don't we just all it Unavailable? gRPC unavailable sounds like an implementation detail